tensorflow/python/ops/image_ops_impl.py

```diff
@@ -640,6 +640,20 @@ def transpose(image, name=None):
 
   Raises:
     ValueError: if the shape of `image` not supported.
+
+  Usage Example:
+    ```python
```

Comment:
Please remove ```python and ``` (backticks) here and everywhere.

You can check out the instructions at https://www.tensorflow.org/community/contribute/docs_ref

Reply:
all removed